worker/appm/conversion/gateway.go

```diff
@@ -139,16 +139,15 @@ func (a *AppServiceBuild) Build() (*v1.K8sResources, error) {
 		}
 		ports, pp, err = a.CreateUpstreamPluginMappingPort(ports, pluginPorts)
 	}
-	if err != nil {
-		return nil, fmt.Errorf("create upstream port error, %s", err.Error())
-	}
+
 	var services []*corev1.Service
 	var ingresses []*extensions.Ingress
 	var secrets []*corev1.Secret
 	if ports != nil && len(ports) > 0 {
 		for i := range ports {
 			port := ports[i]
 			if *port.IsInnerService {
+				services = append(services, a.createKubernetesNativeService(port))
```

Comment:
Create the service repeatedly；
Not based on the governance model；

Reply:
fixed